src/Marten.AsyncDaemon.Testing/event_projections_end_to_end.cs

```diff
@@ -127,6 +127,7 @@ private async Task CheckExpectedResults(IDocumentSession session)
         var events = (await session.Events.QueryAllRawEvents().ToListAsync());
         var travels = events.OfType<Event<Travel>>().ToDictionary(x => x.Id);
 
+        distances.Count.ShouldBe(travels.Count);
```

Comment:
looks like this is why the tests have passed.. `distances` collection was always empty (because `WaitForNonStaleProjectionDataAsync` method didn't await for shards to start), so iteration 
`foreach (var distance in distances)`  below was not executed and nothing was asserted